sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -0,0 +1,25 @@
+[id="upgrade-sandboxed-containers"]
+= Upgrade {sandboxed-containers-first}
+include::modules/common-attributes.adoc[]
+:context: upgrade-sandboxed-containers
+
+toc::[]
+
+[id="sandboxed-containers-upgrade-operator"]
+== Upgrade {sandboxed-containers-operator}
+
+You can upgrade the Operator manually or automatically using the OpenShift Lifecycle Manager (OLM). You can select manual or automatic during the initial deployment. In the context of manual upgrades, the web console shows available updates which can be installed by the cluster administrator.
+
+[id="sandboxed-containers-upgrade-artifacts"]
+== Upgrade the {sandboxed-containers-first} artifacts
+
+The {sandboxed-containers-first} artifacts are deployed onto the cluster using {op-system-first} extensions.
+
+//Update the Red Hat CoreOS extension with the updated link once PR is approved for the Understanding section.
+
+The {op-system} extension `sandboxed containers` contains the required components to run Kata containers such as the Kata containers runtime, the hypervisor QEMU-kiwi and its dependencies. The extension is upgraded when upgrading the cluster to a new release of {product-title}.
+
+
+.Additional resources
```

Comment:
Consider placing this section after the first section. (I think it relates more to that section.)